server/models/ViewRouteSentenceAndOffenceViewModel.ts

```diff
@@ -56,20 +55,13 @@ export default class ViewRouteSentenceAndOffenceViewModel {
   }
 
   public rowIsSdsPlus(sentence: PrisonApiOffenderSentenceAndOffences, offence: PrisonApiOffenderOffence): boolean {
-    const input =
+    const oldUserInputForSDSPlus =
       this.userInputs &&
       this.userInputs.sentenceCalculationUserInputs.find((it: CalculationSentenceUserInput) => {
         return it.offenceCode === offence.offenceCode && it.sentenceSequence === sentence.sentenceSequence
       })
-    return input && input.userChoice
-  }
-
-  public lastUserQuestion(): CalculationQuestionTypes {
-    if (this.userInputs) {
-      const userInputTypes = CalculationQuestionTypes.getOrderedQuestionTypesFromInputs(this.userInputs)
-      return userInputTypes[userInputTypes.length - 1]
-    }
-    return null
+    const isUserIdentifiedSDSPlus = oldUserInputForSDSPlus && oldUserInputForSDSPlus.userChoice
+    return isUserIdentifiedSDSPlus || offence.indicators?.includes('PCSC/SDS+')
```

Comment:
and can this be isPcscSdsPlus?

Reply:
I don't think these have tests either? If I'm understanding the code correctly it should show an SDS+ badge on the sentence and offence page in view and calc journeys?

Reply:
That would be correct - i'll add some...